src/main/java/gov/cms/mat/cql_elm_translation/utils/HumanReadableUtil.java

```diff
@@ -124,4 +128,47 @@ public static String getPopulationDescription(Measure measure, String population
     }
     return sb.toString();
   }
+
+  public static String getMeasureObservation(Measure measure) {
```

Comment:
can we refactor / rename this to `getMeasureObservationDescriptions` to better indicate what we're fetching are the descriptions, not the whole measure observation object